### ISSUE TITLE
Move to a better place!

### ISSUE THREAD

**Issue by soheilkhanalipur** [CookieJarApps/SmartCookieWeb-Preview#41]
Unfortunately, I have to say that you have put this program in a bad place! GitHub!
This app deserves places like: Codeberg, Gitea, GitLab (Instances), and wherever better…
GitHub boycotts users, removes apps, and actually says "Whatever I want is going to be saved and whatever I don't need Is going to be deleted." The app has not paid attention to users' requests for RTL support for years, and says this is not a priority!
Free software should be free somewhere, which GitHub does not have!
Aren't these enough to leave GitHub?

**Comment by CookieJarApps** [comment]
Thanks for your suggestion. I understand the concerns about GitHub - I used to have a self-hosted GitLab instance in parallel to the GitHub repo for SmartCookieWeb, but most issues continued to be opened here, and that combined with the cost of hosting the instance made it a bit difficult to justify.

However, since that was a while ago, and cost isn't an issue now, I'll set up either a GitLab or Gitea instance later today, see over the next month how it goes, and possibly make the decision to switch over.

**Comment by CookieJarApps** [comment]
Done! - https://vcs.cookiejarapps.com/

**Comment by soheilkhanalipur** [comment]
You're Great!!
